api/v1alpha1/scopeinstance_types.go

```diff
@@ -37,8 +37,19 @@ type ScopeInstanceSpec struct {
 type ScopeInstanceStatus struct {
 	// INSERT ADDITIONAL STATUS FIELD - define observed state of cluster
 	// Important: Run "make" to regenerate code after modifying this file
+
+	// Conditions represent the latest available observations of an object's state
+	Conditions []metav1.Condition `json:"conditions,omitempty" patchStrategy:"merge" patchMergeKey:"type" protobuf:"bytes,1,rep,name=conditions"`
 }
 
+const (
+	TypeScoped = "Scoped"
```

Comment:
Do we think that `Scoped` is a good name for the condition type here? Would something like `Instantiated` or `Applied` work better?

Reply:
I'm not 100% sure. Scoped sounds find to me. I think we could change it if it's not right in a follow-on PR if needed.